### PR TITLE
Template description

### PR DESCRIPTION
**Pull request by alexandrelissowski**

### CODE REVIEW

File: content/md/supplier-onboarder/dwl-product-import-tpl.md [hunk 1]
```diff
@@ -17,3 +18,19 @@ Here is how to get them:
...
 This template file can easily be imported in your Onboarder to add many products at once.
+
+## File format
+The template contains 3 lines in the header:
+1. attribute code, it allows the Onboarder to read the file when importing it
+1. attribute labels, it is the name of the attribute given by the retailer
+1. requirement, it shows you what attributes are required for [product completion](update-products.html#the-attribute-panel)
+
+:::warning
+Only line bellow the third line will be read at the import. If you change the file and add products before this line, they won't be imported.
```
Comment (bgouraud) [inline content/md/supplier-onboarder/dwl-product-import-tpl.md:29]: bellow --> below
at the import --> at import

File: content/md/supplier-onboarder/dwl-product-import-tpl.md [hunk 2]
```diff
@@ -17,3 +18,19 @@ Here is how to get them:
 1. Choose your preferred format: one file per family or one file with all families, in `CSV` or `XLSX`
 
 This template file can easily be imported in your Onboarder to add many products at once.
+
+## File format
+The template contains 3 lines in the header:
+1. attribute code, it allows the Onboarder to read the file when importing it
```
Comment (bgouraud) [inline content/md/supplier-onboarder/dwl-product-import-tpl.md:24]: attribute code: Unique attribute identifier. Allows the Onboarder to know to which attribute the data refers to when importing.

File: content/md/supplier-onboarder/dwl-product-import-tpl.md [hunk 3]
```diff
@@ -17,3 +18,19 @@ Here is how to get them:
...
+1. attribute labels, it is the name of the attribute given by the retailer
```
Comment (bgouraud) [inline content/md/supplier-onboarder/dwl-product-import-tpl.md:25]: attribute labels: Human-readable designation of the attribute code above given by your retailer.

File: content/md/supplier-onboarder/dwl-product-import-tpl.md [hunk 4]
```diff
@@ -17,3 +18,19 @@ Here is how to get them:
...
+1. requirement, it shows you what attributes are required for [product completion](update-products.html#the-attribute-panel)
```
Comment (bgouraud) [inline content/md/supplier-onboarder/dwl-product-import-tpl.md:26]: requirement status of the attribute: Shows you the status of the attribute (required, optional) for the [product completion](update-products.html#the-attribute-panel)

File: content/md/supplier-onboarder/dwl-product-import-tpl.md [hunk 5]
```diff
@@ -17,3 +18,19 @@ Here is how to get them:
...
+
+:::warning
+Only line bellow the third line will be read at the import. If you change the file and add products before this line, they won't be imported.
+:::
+
+:::info
+The excel file (XLSX) contains drop-downs for single select attributes. Each selection option is displayed with the option label and the option code between brackets. As for the attribute code line, the option code is necessary to read the file during import.
```
Comment (bgouraud) [inline content/md/supplier-onboarder/dwl-product-import-tpl.md:33]: drop-down lists instead of drop-downs.

Comment (bgouraud) [inline content/md/supplier-onboarder/dwl-product-import-tpl.md:33]: You are talking about simple select. How are handled multi-select attributes?

File: content/md/supplier-onboarder/create-products.md [hunk 6]
```diff
@@ -5,8 +5,7 @@ title: Create products
 popular: true
 ---
 
-# Product creation? Piece of cake!
-## Manual creation
+# Manual creation
```
Comment (momoss) [inline content/md/supplier-onboarder/create-products.md:8]:
```suggestion
# Automatic creation
```

**Pull request merged**